kernel/safe_typing.ml

```diff
@@ -902,6 +931,16 @@ let add_module l me inl senv =
   in
   (mp,mb.mod_delta),senv''
 
+(** {6 Interactive sections *)
+
+let open_section ~poly senv =
+  let sections = Section.open_section ~poly senv.sections in
+  { senv with sections }
+
+let close_section senv =
+  (* TODO: implement me when discharging in kernel *)
```

Comment:
Sorry, can you explain what the comment means?

Reply:
This is currently a no-op, but when discharging is implemented in the kernel in a next PR, something will have to be implemented here.